test-specifications/crypto-service/accountCreateTransaction.md

```diff
@@ -0,0 +1,439 @@
+# AccountCreateTransaction - Test specification
+
+## Description:
+This test specification for AccountCreateTransaction is to be one of many for testing the functionality of the Hedera SDKs. The SDK under test will use the language specific JSON-RPC server return responses back to the test driver.
+
+## Design:
+Each test within the test specification is linked to one of the properties within AccountCreateTransaction. Each property is tested with a mix of boundaries. The inputs for each test are a range of valid, minimum, maximum, negative and invalid values for the method. The expected response of a passed test can be a correct error response code or seen as the result of node queries. A successful transaction (the transaction reached consensus and was applied to state) can be determined by getting a `TransactionReceipt` or `TransactionRecord`, or can be determined by using queries such as `AccountInfoQuery` or `AccountBalanceQuery` and investigating for the required changes (creations, updates, etc.). The mirror node can also be used to determine if a transaction was successful via its rest API. Error codes are obtained from the response code proto files.
+
+**Transaction properties:**
+
+https://docs.hedera.com/hedera/sdks-and-apis/sdks/accounts-and-hbar/create-an-account
+
+**CryptoCreate protobufs:**
+
+https://github.com/hashgraph/hedera-protobufs/blob/main/services/crypto_create.proto
+
+**Response codes:**
+
+https://github.com/hashgraph/hedera-protobufs/blob/main/services/response_code.proto
+
+**Mirror Node APIs:**
+
+https://docs.hedera.com/hedera/sdks-and-apis/rest-api
+
+## Initialisation
+
+```jsx
+new AccountCreateTransaction()
+
+//example
+
+const transaction = new AccountCreateTransaction()
+  .setKey(privateKey.publicKey)
+  .setInitialBalance(new Hbar(1000))
+  ...
+```
```

Comment:
Any objections to get rid of this? I don't think it should necessarily be the TCK's responsibility to show how to use the SDK in its documentation; this is what the SDK examples are for.

Reply:
true. no objections from me

Reply:
No objections from me as well 👌🏻

Reply:
Removed